ecs-cli/modules/commands/flags/flags.go

```diff
@@ -250,15 +250,6 @@ func DebugFlag() []cli.Flag {
 	}
 }
 
-func FipsEndpointFlag() []cli.Flag {
```

Comment:
I had originally put this here since we ill need to support FIPS endpoints for other services as well.

Reply:
Yeah, I think we can refactor this again when we add FIPs support for more things. But for now, the function returns the flag with help text, which specifically talks about ECR.